src/swagger.yml

```diff
@@ -3170,6 +3180,38 @@ definitions:
       box:
         $ref: '#/definitions/WineBox'
 
+  ArticleWineWithWineShopTitle:
```

Comment:
Rename to `ArticleWineWithExtendedShopInfo`, name the other `ArticleWineWithBasicShopInfo`.